.circleci/config.yml

```diff
@@ -1389,16 +1389,15 @@ workflows:
       # Yarn Project
       - yarn-project-base:
           requires:
-            - avm-transpiler
```

Comment:
Looks like the transpiler is removed and not added in yarn-project. Is this intentional? (I think it's currently needed to transpile just after compiling the contracts).